themes/scalereal/static/css/style.css

```diff
@@ -3363,4 +3363,148 @@ lottie-player {
 
 .feedback-mobile  .owl-carousel .owl-dots {
   margin-top: -2px;
+}
+
+/* Client Success Stories Section */
+.client-success-stories-section .container .box {
+  padding: auto 98px ; 
+}
+.client-success-stories-section .box::before {
+  content: "";
+  position: absolute;
+  left: 47%;
+  top: 85px;
+  bottom: 85px; 
+  width: 2px;
+  height: 70%;
+  background-color: #cccbcb; 
+}
+.client-success-stories-section .container .box .heading.box-content{
+  margin:100px 25px auto auto;
+}
+.client-success-stories-section .container .box .heading.box-content h2{
+  padding-right: 86px;

```

Comment:
Instead of this use width: 85% to make it responsive. (Adjust width as per design, just for the sake of example I mentioned 85%)